kong/plugins/ldap-auth/access.lua

```diff
@@ -50,17 +50,28 @@ local function ldap_authenticate(given_username, given_password, conf)
 
   sock:settimeout(conf.timeout)
 
-  ok, err = sock:connect(conf.ldap_host, conf.ldap_port)
+  local opts = {}
```

Comment:
We could save this allocation and alleviate GC pressure if `conf.start_tls` isn't truthy.